lib/Extension/LanguageServerPhpstan/Model/Linter/PhpstanLinter.php

```diff
@@ -9,13 +9,16 @@
 use Phpactor\Extension\LanguageServerPhpstan\Model\PhpstanProcess;
 use Phpactor\LanguageServerProtocol\Diagnostic;
 use Phpactor\TextDocument\TextDocumentUri;
+
 use function Safe\tempnam;
 use function Safe\file_put_contents;
 
 class PhpstanLinter implements Linter
 {
-    public function __construct(private PhpstanProcess $process)
-    {
+    public function __construct(
+        private PhpstanProcess $process,
+        private ?bool $disableTmpFile = null,
```

Comment:
can set this to `false` by default and remove the nullable?

Reply:
Good point. 

Done in https://github.com/phpactor/phpactor/pull/2764/commits/6dfab15e86e405a73da2399cb154c59654b69a1d and confirmed working.